test/sign.test.ts

```diff
@@ -16,7 +16,7 @@ import { SAMPLE_ORDER } from "./testHelpers";
 const patchedSignMessageBuilder = (key: SigningKey) => async (
   message: string,
 ): Promise<string> => {
-  // Reproducing `@ethersproject/wallet/src.ts/index.ts` sign message bahaviour
+  // Reproducing `@ethersproject/wallet/src.ts/index.ts` sign message behavior
```

Comment:
Yuk, US spelling! Why not Canadian/British spelling?

Reply:
Sorry, that was accidental.

Reply:
I was just kidding :stuck_out_tongue:. I'm indifferent either way.

Reply:
Ill change it back someday.